src/main/kotlin/com/launchdarkly/intellij/toolwindow/LinkNodeRoot.kt

```diff
@@ -0,0 +1,83 @@
+package com.launchdarkly.intellij.toolwindow
+
+import com.intellij.ide.projectView.PresentationData
+import com.intellij.ui.treeStructure.SimpleNode
+import com.launchdarkly.intellij.settings.LDSettings
+
+class LinkNodeRoot(private val settings: LDSettings) :
+    SimpleNode() {
+    private var myChildren: MutableList<SimpleNode> = ArrayList()
+
+    override fun getChildren(): Array<SimpleNode> {
+        if (settings.project != "" && settings.environment != "") {
+            myChildren.add(
+                LinkNodeBase(
+                    "Flags",
+                    "${settings.baseUri}/${settings.project}/${settings.environment}/features"
+                )
+            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Segments",
+                    "${settings.baseUri}/${settings.project}/${settings.environment}/segments"
+                )
+            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Users",
+                    "${settings.baseUri}/${settings.project}/${settings.environment}/users"
+                )
+            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Debugger",
+                    "${settings.baseUri}/${settings.project}/${settings.environment}/debugger"
+                )
+            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Experiments",
+                    "${settings.baseUri}/${settings.project}/${settings.environment}/experiments"
+                )
+            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Audit Log",
+                    "${settings.baseUri}/${settings.project}/${settings.environment}/audit"
+                )
+            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Flag Comparison",
+                    "${settings.baseUri}/${settings.project}/${settings.environment}/features/compare"
+                )
+            )
+            // TODO: Fix selector so we can uncomment this
+//            myChildren.add(
+//                LinkNodeBase(
+//                    "Open Flag Environment Overview",
+//                    "${settings.baseUri}/${settings.project}/${settings.environment}/features/"
+//                )
+//            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Open Documentation",
+                    "https://docs.launchdarkly.com"
+                )
+            )
+            myChildren.add(
+                LinkNodeBase(
+                    "Open API Documentation",
+                    "https://apidocs.launchdarkly.com"
+                )
+            )
+        }
+
+        return myChildren.toTypedArray()
```

Comment:
In the beginning, when the plugin is not configured (no api key), the quick links panel should help the user configure the plugin. @darrinlin any ideas here? I have two thoughts:
1. Display the [Account settings link](https://app.launchdarkly.com/settings/authorization) to get the api key so users can login and get an api key.
2. Or simply display a string saying something like "Please configure the plugin with an api key"

Reply:
That can be tackled as part of a larger followup PR. This is getting it landed in time for GA.